.eslintrc.json

```diff
@@ -79,10 +79,7 @@
         "key-spacing": "off",
         "keyword-spacing": "off",
         "line-comment-position": "off",
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
+        "linebreak-style": 0,
```

Comment:
Hmm - is there a reason for turning off this check?

Reply:
yeap, I wrote in general comment to the PR that without turning it off, linting on windows will always fail with `Expected linebreaks to be 'LF' but found 'CRLF'`

Reply:
https://stackoverflow.com/questions/37826449/expected-linebreaks-to-be-lf-but-found-crlf-linebreak-style

Reply:
Do you know if to use this workaround (https://stackoverflow.com/a/65519055) the whole .eslintrc.json file needs converting to .eslintrc.js ?

Reply:
Ah, it looks like you can do a `.eslintrc.js` file like:

```js
module.exports = {
    "extends": "./.eslintrc.json",
    "rules": {
        // override default options
       'linebreak-style': ['error', process.platform === 'win32' ? 'windows' : 'unix']
    }
};
```

Reply:
Looks like we're nearly there! Thanks for continuing to work on this.

Reply:
@MikeRalphson I pushed this new js file and looks good here https://github.com/derberg/mdv/pull/1 - all checks passed